volumes/miovision/sql/function/function-identify-zero-counts.sql

```diff
@@ -0,0 +1,86 @@
+CREATE OR REPLACE FUNCTION miovision_api.identify_zero_counts(
+    start_date timestamp,
+    end_date timestamp,
```

Comment:
What kind of `start_date`, `end_date` values will this function be run with? A one-day interval? Something else?

Reply:
Should these be called `_date` when they are actually timestamps?

Reply:
Yes, a one day interval run by the daily Airflow DAG included in this PR. 
I'll consider changing the variables names, but this is pretty consistent with the other Miovision function conventions.

Reply:
It makes sense if they'll always be date values, essentially.